src/easy/handler.rs

```diff
@@ -2921,6 +2922,8 @@ extern fn ssl_ctx_cb<H: Handler>(_handle: *mut curl_sys::CURL,
                                  ssl_ctx: *mut c_void,
                                  data: *mut c_void) -> curl_sys::CURLcode {
     let res = panic::catch(|| unsafe {
+        windows::add_certs_to_context(ssl_ctx);
```

Comment:
Perhaps we should only do this in the default trait method? That way it could be entirely overridden if need be I think?